index.html

```diff
@@ -164,8 +164,8 @@ <h3>Installing</h3>
 
             <p><pre><code class="js">{
     "require": {
-        "nesbot/carbon": "2.27.0 as 1.39.0"
-        "kylekatarnls/laravel-carbon-2": "^1.0.0"
+        "nesbot/carbon": "2.27.0 as 1.39.0",
+        "kylekatarnls/laravel-carbon-2": "^1.0.0",
```

Comment:
There is no trailing comma support in JSON.

Reply:
sorry sir. updated.